### PR TITLE
Update DG user stories

### LINKED ISSUE #201: Update DG - User Stories
Update DG user stories to match current implementation

### PR DESCRIPTION
**Pull request by sannie-beep**
fixes #201

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
-| `* *`    | user        | undo the most recent command                                                               | easily undo commands if I key in the wrong information                               |
+| Priority | As a …​    | I want to …​                                                                     | So that I can …​                                                                     |
+|----------|------------|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| `* * *`  | user       | add a new person                                                                 | store a new person in my contact list                                                |
+| `* * *`  | user       | add details to contacts                                                          | store details of people in my contact list                                           |
+| `* * *`  | researcher | add multiple tags to participants                                                | tag contacts to multiple study groups                                                |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:242]:
```suggestion
| `* * *`  | researcher | add multiple tags to contacts                                                | tag participants to multiple study groups                                                |
```

Comment (PrishaVP) [inline docs/DeveloperGuide.md:242]:
```suggestion
| `* * *`  | researcher | add multiple tags to participants                                                | tag participants to multiple study groups                                                |
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| `* * *`  | user       | delete contacts                                                                  | remove old/ contacts that I no longer need                                           |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:243]: remove the random /

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| `* * *`  | user       | work on a clean, user-friendly *UI*                                              | navigate the platform with ease                                                      |
+| `* * *`  | user       | list all contacts                                                                | view my list of contacts                                                             |
+| `* * *`  | user       | see usage instructions                                                           | refer to instructions when I forget how to use the app                               |
+| `* *`    | user       | be given a prompt on what format to enter details                                | easily use commands without having to memorise the accepted format for the CLI entry |
+| `* *`    | user       | edit contacts                                                                    | update details of my contacts                                                        |
+| `* * *`  | researcher | delete participant information in bulk                                           | quickly clear old groups of participants                                             |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:252]:
```suggestion
| `* * *`  | researcher | clear participant information in bulk                                           | quickly clear old groups of participants                                             |
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| Priority | As a …​    | I want to …​                                                                     | So that I can …​                                                                     |
+|----------|------------|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
+| `* * *`  | user       | add a new person                                                                 | store a new person in my contact list                                                |
+| `* * *`  | user       | add details to contacts                                                          | store details of people in my contact list                                           |
+| `* * *`  | researcher | add multiple tags to participants                                                | tag participants to multiple study groups                                                |
+| `* * *`  | user       | delete contacts                                                                  | remove old/ contacts that I no longer need                                           |
```
Comment (sannie-beep) [inline docs/DeveloperGuide.md:243]:
```suggestion
| `* * *`  | user       | delete contacts                                                                  | remove old contacts that I no longer need                                           |
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| `* *`    | user       | clear all entries                                                                | efficiently restart my progress                                                      |
+| `* *`    | user       | be prompted for confirmation before clearing my contacts                         | ensure that I do not make a mistake when doing a large irreversible delete           |
+| `* * *`  | researcher | search for participants based on criteria(s) (i.e., study group and/or other details) | quickly locate specific groups of participants                                       |
+| `* * *`  | researcher | add and delete tags to and from a specific exisitng participant                  | effectively update the participants' study group for organization                    |
+| `* * *`  | user       | filter contacts by any attribute                                                 | view my contacts by specific groups                                                  |
+| `* * *`  | researcher | find participants usinf multiple criteria                                        | view specific participants of interest                                               |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:258]: spelling

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| `* *`    | researcher | export contacts emails (or other details) into easy to copy-paste format         | copy the details (like emails) into other places easily                              |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:259]: only emails

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -235,38 +235,30 @@ Thus, *ResearchRoster* allows researchers to save time, effort and energy whilst
...
+| `* *`    | researcher | assign participants randomly into sample groups                                  | easily obtain samples for experiments                                                |
```
Comment (PrishaVP) [inline docs/DeveloperGuide.md:260]:
```suggestion
| `* *`    | researcher | assign participants randomly into study groups                                  | easily obtain randomized study groups for experiments                                                |
```

**Comment by PrishaVP** [comment]
other than the 2 minor nits, LGTM

**Pull request merged**